app/models/viewmodels/errors/ErrorInterpreter.scala

```diff
@@ -99,7 +100,18 @@ trait ErrorInterpreter {
 
   def generateFieldTable(error: ErrorInstance)(implicit messages: Messages): Seq[Html] =
     error.fieldsInvolved.map { field =>
-      val fieldName = messages(field.pointer.messageKey, field.pointer.sequenceArgs: _*)
+      val fieldName = field.pointer.pattern match {
+        // Because APCs occupy the same document location as PCs, the sequence needs correcting so that any given APC is not displayed as "APC n + 1" to the user
+        case `pointerToAdditionalProcedureCodes` =>
+          messages(
+            field.pointer.messageKey,
+            field.pointer.sequenceArgs.head,
+            field.pointer.sequenceArgs.last.toIntOption.fold("")(x => (x - 1).toString)
+          )
+        case _ =>
+          messages(field.pointer.messageKey, field.pointer.sequenceArgs: _*)
+
+      }
```

Comment:
Do you still need this now? :thinking:

Reply:
Yes, this is for the CDS12056 error that occurs with an additional procedure code pointer, which we previously didn't have.

Reply:
Well, technically it would be for any error code that comes through with an APC pointer.